src/main/resources/logback.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <layout class="ch.qos.logback.classic.PatternLayout">
+            <Pattern>
+                %d{yyyy-MM-dd HH:mm:ss} %-5level %logger{36} - %msg%n
+            </Pattern>
+        </layout>
+    </appender>
+
+    <logger name="org.springframework" level="info" additivity="false">
+        <appender-ref ref="STDOUT" />
+    </logger>
+
+    <logger name="ru.timur.learning" level="debug" additivity="false">
+        <appender-ref ref="STDOUT" />
+    </logger>
```

Comment:
afaic we can just leave `<logger name="ru.timur.learning" level="debug"/>` with the same effect

Reply:
Refactored in commit 1ac7ffc